source/_docs/configuration/templating.markdown

```diff
@@ -98,6 +98,8 @@ In your automations, you could then reuse this macro by importing it:
 
 {% endraw %}
 
+Important Note: The return value from a macro will always be converted to a string - when being used in a YAML template
```

Comment:
- We have a format for important notes, see also: https://developers.home-assistant.io/docs/documenting/create-page#html
- The sentence doesn't end with a full stop.